package.json

```
@@ -47,13 +47,13 @@
     "start": "gatsby develop",
     "format": "prettier --write \"**/*.{ts,tsx,js,scss}\" && npm run lint:fix",
     "format-check": "prettier --check \"**/*.{ts,tsx,js,scss}\"",
-    "jest": "jest",
-    "update-snapshot": "jest --updateSnapshot",
-    "test": "npm run format && npm run format-check && npm run lint && npm run jest",
-    "test-watch": "npm run jest --watch",
-    "test-ci": "npm run test --coverage && codecov",
+    "update-snapshot": "npm run test -- --updateSnapshot",
+    "pretest": "npm run format && npm run format-check && npm run lint",
+    "test": "npx jest",
```

```suggestions
    "test": "jest",
```
Since we know this will be available in the path can't we avoid the call to npx?

---

Ah sorry, should have called that out. We certainly can; I tend to use it as a default, I've seen odd behavior where you need to either use `npx` or point directly to `./node_modules/.bin/*` - but I can't give specifics, so:

```suggestion
    "test": "jest",
```